addon/components/frost-list.js

```diff
@@ -35,7 +35,9 @@ export default Component.extend({
       PropTypes.EmberObject,
       PropTypes.object
     ])),
+    isLoading: PropTypes.bool,
     itemExpansion: PropTypes.EmberComponent,
+    loadingType: PropTypes.string,
```

Comment:
I'd like this to be change to a component closure rather than relying directly on `frost-loading`

So the interface would be

```
{{frost-list
  loading=(component 'frost-loading')
  ...
}}
```